README.md

```diff
@@ -52,6 +52,17 @@ Then simply add a new powershell task to download the exe, and run it to update
 
 The CLI also supports creating or updating manifests with multiple installer URLs. You can either create new manifests with multiple installer nodes using the [New Command](doc/new.md) or update existing manifests with multiple installer URLs using the [Update Command](doc/update.md). 
 
+## Using your GitHub Personal Access Token
+
+When [creating your own GitHub Personal Access Token (PAT)](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) to be used with WingetCreate, make sure you verify the following steps are completed:
+
+- [Authorizing your personal access token for use with SAML single sign-on](https://docs.github.com/en/github/authenticating-to-github/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)
```

Comment:
Are you sure everyone needs to do this, or just us MSers?

Reply:
Oops, you're right. Just us MSers. Deleted.